lesspipe.sh

```diff
@@ -113,6 +113,11 @@ filetype () {
     set "$tmpdir/file" "$2"
     name="$filen"
   fi
+  if [[ ("$name" = *.br || "$name" = *.bro || "$name" = *.tbr) ]]; then
```

Comment:
if the detection is made only on filename the dumping the file with dd is useless, should your code should be moved up